cogs/utilities.py

```diff
@@ -43,7 +43,7 @@ async def checkRoles(self, ctx: Context):
 e.g. `help-1` `help-1` and not `help 1` `help-1`.
                 
 Optionally:
-You can hide the help channels for `@everyone` and `verified`.
+You can hide the help channels for `@everyone` and `@verified`.
```

Comment:
The reason `@everyone` has a @ and not verified is because the role `@everyone` is exactly that, while the role for `verified` is `verified` and not `@verified`. The @ is just there to tag people with the role. 
![image](https://user-images.githubusercontent.com/16852963/95361249-653c6080-08c4-11eb-908c-8f2914dabf5c.png)